script.js

```diff
@@ -1,4 +1,299 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+var cardName = [
+  "ace",
+  "two",
+  "three",
+  "four",
+  "five",
+  "six",
+  "seven",
+  "eight",
+  "nine",
+  "ten",
+  "jack",
+  "queen",
+  "king",
+];
+
+var suits = ["Hearts", "Diamonds", "Clubs", "Spades"];
+
+// To create an array to store the cards drawn by player >> playerHand [i]
+// To create a function that totals up the value of the cards in the array
+// To win, playerTotal == 21 or more than dealer but still below 21
+
+// Game Mode
+var mode = "hit";
+function hit() {
+  playerDraws();
+  computerDraws();
+  return checkPlayerCards();
+}
+
+// Player and computer hand as an array to track cards drawn
+var playerHand = [];
+var computerHand = [];
+
+function playerDraws() {
+  var card = shuffledDeck.pop();
+  playerHand.push(card);
+}
+
+function aceOneOrEleven() {
+  if (handTotal(playerHand) <= 11) {
+    for (var i = 0; i < playerHand.length; i += 1) {
+      if (playerHand[i].rank == 1) {
+        playerHand[i].rank = 11;
+        i = playerHand.length;
+      }
+    }
+    return checkPlayerCards();
```

Comment:
It's odd that you are returning the same thing for both if and else statements. You probably can just return it once at the end of the function